napari_aicssegmentation/view/workflow_select_view.py

```diff
@@ -75,6 +77,12 @@ def setup_ui(self):
         ]
         for widget in widgets:
             layout.addWidget(widget)
+
+        # Add workflow buttons
+        engine = WorkflowEngine()
+        self.workflow_buttons = self._load_workflows(engine.workflow_definitions)
+        layout.addWidget((self.workflow_buttons))
+
         self._add_step_3_layout(layout, enabled=False)
```

Comment:
Actually, I think the buttons should be added from inside `_add_step_3_layout`, because the `enabled` param for `_add_step_3_layout` can then control all the things that need to be disabled until the user selects a layer and a channel. (i.e., the step 3 instructions and the column labels ("image input" and "segmentation output") all need to be disabled or enabled in sync with the buttons)). Let me know if that doesn't make sense.